### ISSUE TITLE
Command and args customize in sidecar?

### ISSUE THREAD

**Issue by kost2191** [zalando/postgres-operator#2439]
Please, answer some short questions which should help us to understand your problem / question better?

- **Which image of the operator are you using?**  registry.opensource.zalan.do/acid/postgres-operator:v1.10.1
- **Where do you run it - cloud or metal? Kubernetes or OpenShift?** AWS K8s 
- **Are you running Postgres Operator in production?** yes
- **Type of issue?**feature request

We have added some sidecars (exporters, pgbouncer), but we cannot change its command or args via kind:Postgresql
Can you add this feature for customizing "command" and "args"?

**Comment by mhaley-tignis** [comment]
Duplicate of https://github.com/zalando/postgres-operator/issues/2448
PR here https://github.com/zalando/postgres-operator/pull/2449

**Comment by tabbyl21** [comment]
Yep I validated this PR works: #2449 . Wondering what else I need to do to get the PR merged :)

**Comment by petrushinvs** [comment]
Strong Vote for this feature. Any plans to merge?

**Comment by ItielOlenick** [comment]
Any updates? Seems like a very basic requirement
